queue/queue.js

```diff
@@ -26,7 +26,6 @@ var queueApp = angular.module("queueApp", ["ngMaterial", "ngMessages"])
 					appointment.paperworkComplete = appointment.timeReturnedPapers != null;
 					appointment.preparing = appointment.timeAppointmentStarted != null;
 					appointment.ended = appointment.timeAppointmentEnded != null;
-					appointment.name = appointment.firstName + " " + appointment.lastName;
```

Comment:
Whoops, this needs to stay here, this is used in the search functionality. My bad.